MAINTAINING.md

```diff
@@ -48,6 +48,9 @@ git config --global alias.logs 'log --show-signature'
 
 # Pull requests
 
+- **do not** push any `Merge`, `Squash & Merge`, etc. buttons on the website!
+  The only allowed way of merging is locally, since otherwise merge will not
+  be signed, and websites can fairly well mess things up.
```

Comment:
Uh thanks, didn't have that in mind! Argh, this damn (good actually :)) gpg signing rule. Now I understand the problem with #3580. Unfortunately we cannot deactivate the "merge" button on Github. Thanks for deactivating it on Reviewable.